app/models/user.rb

```diff
@@ -93,12 +93,8 @@ def github_client
     @github_client
   end
 
-  def subject_client
-    Octokit::Client.new(access_token: subject_token, auto_paginate: true)
-  end
-
-  def subject_token
-    app_token || effective_access_token
+  def app_installation_client
```

Comment:
The ONLY usage of this is now in `#sync_app_installation_access` in this model.

I renamed it in an attempt to make things more clear, but I am not sure if this worked lol

Reply:
I think this makes sense, as it's only used in `sync_app_installation_access` then we can move this method `private` to the user class

Reply:
I was wrong about this, I am using it in `Notification#github_client` so I think it needs to stay public